### ISSUE TITLE
Update via docker: Migration Error 1064

### ISSUE THREAD

**Issue by m50434** [go-vikunja/api#103]
### Description

Hi,
I just updated Vikunja API ("latest" in Docker) and im getting the following error when starting the container:

migration/Migrate 052 Migration failed: migration 20221113170740 failed: Error 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near 'COLUMN `list_id` TO `project_id`' at line 1

Any ideas what to do?

### Vikunja Frontend Version

latest

### Vikunja API Version

latest

### Can you reproduce the bug on the Vikunja demo site?

No

**Comment by kolaente** [comment]
Which mariadb version are you using?

**Comment by m50434** [comment]
mariadb 10.3.32 - 1040 (Im using Synology NAS. The system says its the newest version)

**Comment by kolaente** [comment]
Looks like your mariadb version is too old. You need at least 10.5.2 to run that migration. 

10.3.32 was released in late 2021, you'll probably want to upgrade that anyway (even if Synology does not offer a newer version directly).

For Vikunja, you could switch to sqlite, probably the resource-friendlier option for a nas anyway.

**Comment by m50434** [comment]
Thanks for your fast reply.

Im sorry. I was using an old Firmware for the Synoloy as well. Therefore I couldnt update MariaDB.

Anyway, now I updated MariaDB (its now 10.11) and the vikunja-api seems to be finde (the log says the migration was succesful and there are no more errors.

Unfortunately something is wrong with the frontend:
If Im try to access vikunja I got a "502 Bad Gateway error".

The log of the frontend is "/bin/sh: /run.sh: not found".
